lib/wrappers/transform-node.ts

```diff
@@ -18,7 +20,15 @@ export default class TransformNodeWrapper extends NodeWrapper {
       outputPath: this.outputPath,
       cachePath: this.cachePath,
     });
-    this.callbackObject = this.nodeInfo.getCallbackObject();
+    //adding ts-ignore for now. Should be removed later when new version of broccoli-node-api is released
+    // @ts-ignore
+    if (this.nodeInfo.fsFacade) {
+      this.callbackObject = this.nodeInfo.getCallbackObject();
+      this.callbackObject.input = new FSMerger(this.nodeInfo.inputNodes).fs,
+      this.callbackObject.output = OutputWrapper(this);
+    } else {
+      this.callbackObject = this.nodeInfo.getCallbackObject();
```

Comment:
Why isn't this done unconditionally above this `if`? It seems like both sides need this...